app/models/search/bookmark_indexer.rb

```diff
@@ -84,7 +84,7 @@ def document(object)
     json_object = object.as_json(
       root: false,
       only: [
-        :id, :created_at, :bookmarkable_type, :bookmarkable_id, :user_id,
+        :id, :created_at, :bookmarkable_type, :bookmarkable_id,
```

Comment:
This `user_id` was duplicate with the one in the merge call later (as_json uses string keys, the merge uses symbols, so it wasn't getting overwritten by the merge). Something changed in the json serialization for Elasticsearch so it wasn't getting rid of this duplicate `"user_id": nil` anymore, so deleting it here is the fix.